get-mods.sh

```diff
@@ -92,3 +92,9 @@ get_mod pdfexport
 get_mod reporting
 get_mod vspheredb
 get_mod x509
+if [ "$MODE" = "release" ]; then
+  # Icinga Kubernetes Web does not yet have a release, but should also be included:
+  # Be careful when adding code after the following line, as MODE will not be reset.
+  $MODE="snapshot"
+fi
+get_altname icinga-kubernetes-web kubernetes
```

Comment:
The following should give you the (mostly[^1]) same behavior without cluttering the variable:
```sh
MODE=${MODE/release/snapshot} get_altname icinga-kubernetes-web kubernetes
```

[^1]: It would also replace a substring not only a full match.

Reply:
Since the code is subject to remove anyway, I didn't care about the aesthetics. But true, this looks better.

Reply:
It's not only about the looks, it actually fixes this:
> Be careful when adding code after the following line, as MODE will not be reset.

Reply:
> It's not only about the looks, it actually fixes this:
> 
> > Be careful when adding code after the following line, as MODE will not be reset.

Shame on me. Yes.